bin/lenscompile.js

```diff
@@ -0,0 +1,108 @@
+#! /usr/bin/env node
+
+/**
+ * bin/lenscompile.js
```

Comment:
camelCase preferred for file names, so this file should be named "lensCompile.js"